web/src/redux/reducers/apd.test.js

```diff
@@ -708,6 +711,7 @@ describe('APD reducer', () => {
 
     expect(apd(state, { type: REMOVE_APD_YEAR, value: '1741' })).toEqual({
       data: {
+        apdType: APD_TYPE.HITECH,
```

Comment:
could you add test cases for adding a year and removing a year from MMIS

Reply:
Absolutely! didn't realize I skipped that part.